lizmap/app/responses/AbstractLizmapHtmlResponse.php

```diff
@@ -16,6 +16,24 @@ class AbstractLizmapHtmlResponse extends jResponseHtml
 {
     protected $CSPPropName = 'mapCSPHeader';
 
+    protected function outputJsScriptTag($fileUrl, $scriptParams)
```

Comment:
These functions will _implicitly_ add a `preload` tag for each resource loaded with `addJSLink`/`addCssLink`, no ?